src/Utils.h

```diff
@@ -44,13 +44,14 @@ namespace Utils {
 
     template<typename T>
     T rotl(const T x, const int k) {
-	    return (x << k) | (x >> (std::numeric_limits<T>::digits - k));
+        return (x << k) | (x >> (std::numeric_limits<T>::digits - k));
     }
 
     inline bool is7bit(int c) {
         return c >= 0 && c <= 127;
     }
 
+    /* Returns the smallest multiple of b that is larger than a */
```

Comment:
Looking at the code, that's not true?

Reply:
Based on the unit tests (and the name of the function), this is exactly what it does. 
Here are some sample tests:
 ```
EXPECT_EQ(ceilMultiple(13, 5), (size_t)15);
EXPECT_EQ(ceilMultiple(23, 5), (size_t)25);
EXPECT_EQ(ceilMultiple(99, 100), (size_t)100);
EXPECT_EQ(ceilMultiple(149, 10), (size_t)150);
```
Is there a better way to describe it?

Reply:
What does ceilMultiple(5, 5) return?

Reply:
Its five. I added a test for it
EXPECT_EQ(ceilMultiple(5, 5), (size_t)5);
I will change the comment from 
`Returns the smallest multiple of b that is larger than a`
to
`Returns the smallest multiple of b that is larger than or equal to a`